setup.py

```diff
@@ -134,6 +134,6 @@ def find_package_data(where=".", package="", exclude=standard_exclude,
     long_description=LONG_DESCRIPTION,
     platforms=['any'],
     classifiers=CLASSIFIERS,
-    install_requires=['boto>=2.1.0'],
+    install_requires=['boto>=2.31.0'],
```

Comment:
I compared metadata on pypi:

https://pypi.python.org/pypi/boto/2.30.0
https://pypi.python.org/pypi/boto/2.31.0

`2.31.0` is the first one to claim supporting py3